src/FastMoneyTurn.js

```diff
@@ -0,0 +1,33 @@
+import domUpdates from './domUpdates';
+import Turn from './Turn';
+
+class FastMoneyTurn extends Turn {
+  constructor() {
+    super();
+  }
+
+  endTurn(game) {
+    game.currentRound.currentPlayer === game.playerOne ? game.currentRound.currentPlayer = game.playerTwo : game.currentRound.currentPlayer = game.playerOne;
+    if (game.currentRound.currentSurveyAnswers.length === 0 && game.roundCounter < 4) {
+      game.roundCounter++;
+      domUpdates.populateQuestionsAndAnswers(game.currentRound);
+      game.currentRound.startFastMoneyTurn();
+    } 
+    else {
+      game.endGame();
+    }
+  }
+
+  timeoutEndTurn(game) {
+    if (game.roundCounter < 4) {
+      game.roundCounter++;
+    domUpdates.populateQuestionsAndAnswers(game.currentRound);
+    game.currentRound.startFastMoneyTurn();
+    }
+    else {
+      game.endGame();
+    }
+  }
```

Comment:
Curious about the difference between `endTurn` and `timeoutEndTurn` methods?

Reply:
In the fastMoneyRound there's two ways the turn will end: by answering all the questions correctly, or by the timer running out. We haven't implemented the setTimeOut yet, so we're not sure if we would need a new method.